docs/content/guides/author-apps/aws/howto-aws-resources/snippets/dynamodb.bicep

```diff
@@ -0,0 +1,34 @@
+import aws as aws
+
+resource dynamodb 'AWS.DynamoDB/GlobalTable@default'={
+  properties: {
+    TableName:'demotable'
+    BillingMode: 'PAY_PER_REQUEST'
+    AttributeDefinitions: [
+      {
+        AttributeName: 'order_id'
+        AttributeType: 'S'
+      }
+      {
+        AttributeName: 'quantity'
+        AttributeType: 'N'
+      }
+    ]
+    KeySchema: [
+      {
+        AttributeName: 'order_id'
+        KeyType: 'HASH'
+      }
+      {
+        AttributeName: 'quantity'
+        KeyType: 'RANGE'
+      }
+    ]
+    Replicas: [
+     {
+      Region: 'us-west-2'
+     } 
+    ]
+  }
+  alias: 'dynamodb'
```

Comment:
Could we put alias at the top before properties as it serves a similar purpose to name in Azure/Radius resources?